src/Components/Post/PostId.tsx

```diff
@@ -56,20 +62,24 @@ const PostId = () => {
   }, [id])
 
   useEffect(() => {
-    axios(`https://jsonplaceholder.typicode.com/comments`)
+    axios('https://jsonplaceholder.typicode.com/comments')
       .then((response) => {
-        setCommentData(response.data)
+        setCommentData(
+          response.data?.filter(
+            (comment: { postId: any }) => comment.postId === id,
```

Comment:
what is `id`? I would rename it in `postId` so it gives a better context about it